default.ps1

```diff
@@ -78,7 +78,7 @@ task Nuget {
       @("RhinoDSL", "1.0.0"), `
       @("Common.Logging", "3.0.0"), `
       @("Common.Logging.Core", "3.0.0"), `
-      @("FileHelpers", "2.0.0.0") `
+      @("FileHelpers", "3.0.46-beta") `
```

Comment:
I fount it http://docs.nuget.org/create/versioning

You must use: @("FileHelpers", "[3.0.46-beta]")